.circleci/run_docker_build.sh

```diff
@@ -17,18 +17,22 @@ if hash docker-machine 2> /dev/null && docker-machine active > /dev/null; then
     export HOST_USER_ID=$(docker-machine ssh $(docker-machine active) id -u)
 fi
 
-ARTIFACTS="$FEEDSTOCK_ROOT/build_artifacts"
+if [[ "${TEST_CONTAINER:"1"}" != "0" ]]; then
```

Comment:
Locally this works for me but in CI I get an error about an unbound variable. Does someone know how to do that correctly?